content/en/tracing/trace_collection/library_injection_local.md

```diff
@@ -225,6 +225,15 @@ The library injection mechanism for Python only supports injecting the Python li
 - **Solution**: Remove the installation of `ddtrace` if library injection is desired. Otherwise, use the installed library ([see documentation][26]) instead of library injection.
 
 
+#### TLS issues
+
+##### `tls: protocol version not supported`
+
+- **Problem**: Since Cluster Agent v1.20, the API is only served with TLS v1.3 by default. If Kubernetes cluster is configured with TLS v1.2 or older, library injection fails.
```

Comment:
```suggestion
- **Problem**: Since Cluster Agent v1.20, the API is only served using TLS v1.3 by default. If the Kubernetes cluster is configured with TLS v1.2 or older, library injection fails.
```